src/main/java/org/testng/TestNG.java

```diff
@@ -276,10 +278,11 @@ public void initializeSuitesAndJarFile() {
 
         for (String suiteFile : s.getSuiteFiles()) {
             try {
-                Collection<XmlSuite> childSuites = getParser(suiteFile).parse();
+            	 Path rootPath = Paths.get(s.getFileName()).getParent();
```

Comment:
Something seriously wrong with my fork. Let me delete and re create a fork and come back with this change.